hphp/runtime/base/timezone.cpp

```diff
@@ -107,7 +107,11 @@ const timelib_tzdb* timezone_get_tzdb() {
   if (s_tzdb_cache.load() == nullptr) {
     Lock tzdbLock(s_tzdb_mutex);
     if (s_tzdb_cache.load() == nullptr) {
+      #ifdef HAVE_SYSTEM_TZDATA
+      s_tzdb_cache = timelib_zoneinfo("/usr/share/zoneinfo");
```

Comment:
again feel free to suggest an alternate way to do this. I considered defining a constant for this but it'd only be used here so not sure if that's necessary.

I also considered changing line 104 instead